Update: Alerts & Notifications Best Practices Guide (#2517)
getsentry/sentry-docs: src/docs/product/sentry-basics/guides/alert-notifications/index.mdx

description: "Learn more about how to both understand and manage the alerts and notifications that give real-time visibility into errors happening in your code."
---

> Sentry’s alert notifications give real-time visibility into errors happening in your code and the impact on your users regardless of where your code is deployed. Alerts explain why you’re being notified and uncover meaningful and customizable information about the error and its context. In addition, Sentry will notify your project team members about various workflow activities.

Reviewer: Do want to separate alerts from notifications?
```suggestion
Both Sentry's alerts and notifications provide real-time visibility into errors happening in your code and the impact on your users regardless of where your code is deployed. Alerts uncover meaningful and customizable information about an error and its context. In addition, Sentry send notifications to your project team members regarding various workflow activities.
```

## Alerts

> When you create a new project in Sentry, you can choose from a few default alerts or create one later. To create an alert, navigate to **Alerts > New Alert Rule** and choose between a [Metric Alert](/product/sentry-basics/guides/alert-notifications/metric-alerts/) or an [Issue Alert](/product/sentry-basics/guides/alert-notifications/issue-alerts/).

Reviewer: ```suggestion
When you create a new project in Sentry, you can choose from among a few default alerts, or choose to create alerts at a later time. To create an alert, navigate to **Alerts > New Alert Rule**, then select whether to create a [Metric Alert](/product/sentry-basics/guides/alert-notifications/metric-alerts/) or an [Issue Alert](/product/sentry-basics/guides/alert-notifications/issue-alerts/).
```

![Choice between Metric Alert or Issue Alert](create-new-alert-rule.png)

> Metric alerts are usually a higher signal than issue alerts because you know upfront what you're monitoring. In contrast, issue alerts power a triage workflow — you cast a wide net for potential problems and review them as they happen.

Reviewer: I'm not sure what we mean by "higher signal" here - it comes across as jargon - can we simplify?

> Choosing a default alert, such as getting notified only **once** when an error is first seen, is generally useful for catching new errors in real-time, but will not notify you about subsequent instances of the same errors. This means that with only the default alert rule in place, you might be losing visibility into critical issues building up in your code.

Reviewer: ```suggestion
Choosing a default alert, such as getting notified only **once** when an error is first seen, is generally useful for catching new errors in real-time, but means Sentry will not alert you about subsequent instances of the same error. This means that with only the default alert rule in place, you might be losing visibility into critical issues building up in your code.
```
